src/components/MultiHopTrade/components/TradeConfirm/TradeConfirm.tsx

```diff
@@ -346,23 +346,16 @@ const FirstHop = ({
     [swapperName, tradeQuote.recommendedSlippage],
   )
 
-  const networkFeeFiatPrecision = useMemo(
-    () => getHopTotalNetworkFeeFiatPrecision(tradeQuoteStep),
-    [tradeQuoteStep],
-  )
-
-  const protocolFeeFiatPrecision = useMemo(
-    () => getHopTotalProtocolFeesFiatPrecision(tradeQuoteStep),
-    [tradeQuoteStep],
-  )
+  const networkFeeFiatPrecision = selectHopTotalNetworkFeeFiatPrecision(store.getState(), 1)
+  const protocolFeeFiatPrecision = selectHopTotalProtocolFeesFiatPrecision(store.getState(), 1)
```

Comment:
first hop, so index is 0
```suggestion
  const networkFeeFiatPrecision = selectHopTotalNetworkFeeFiatPrecision(store.getState(), 0)
  const protocolFeeFiatPrecision = selectHopTotalProtocolFeesFiatPrecision(store.getState(), 0)
```

Reply:
https://github.com/shapeshift/web/pull/4782/commits/ff03b92e29699e757281d5948db6e783cb48ca52